<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
    <testsuite name="force.apex" timestamp="2019-11-19T17:31:16-06:00" hostname="https://business-force-3201-dev-ed.cs79.my.salesforce.com" tests="1" failures="1"  errors="0"  time="0.19">
        <properties>
            <property name="outcome" value="Failed"/>
            <property name="testsRan" value="1"/>
            <property name="passing" value="0"/>
            <property name="failing" value="1"/>
            <property name="skipped" value="0"/>
            <property name="passRate" value="0%"/>
            <property name="failRate" value="100%"/>
            <property name="testStartTime" value="Nov 19, 2019 5:31 PM"/>
            <property name="testExecutionTime" value="0.19 s"/>
            <property name="testTotalTime" value="0.19 s"/>
            <property name="commandTime" value="3.07 s"/>
            <property name="hostname" value="https://business-force-3201-dev-ed.cs79.my.salesforce.com"/>
            <property name="orgId" value="00D1h0000005VYKEA2"/>
            <property name="username" value="test-vv7smmwe0aeu@example.com"/>
            <property name="testRunId" value="7071h00000jQyI1"/>
            <property name="userId" value="0051h000002oYq0AAE"/>
        </properties>
        <testcase name="shouldCreateAnswersOnResponseCreate" classname="CreateResponseAnswersTest" time="0.19">
            <failure message="System.AssertException: Assertion Failed: Expected: 39, Actual: 37"><![CDATA[Class.CreateResponseAnswersTest.shouldCreateAnswersOnResponseCreate: line 28, column 1]]></failure>
        </testcase>
    </testsuite>
</testsuites>
